doc/source/index.rst

```diff
@@ -33,8 +33,8 @@ See Also
 
 * `MontePy github Repository <https://github.com/idaholab/montepy>`_
 * `MontePy PyPI Project <https://pypi.org/project/montepy/>`_
+* `MCNP 6.3 User Manual <https://www.osti.gov/biblio/1889957>`_ DOI: `10.2172/1889957 <https://doi.org/10.2172/1889957>`_
 * `MCNP 6.2 User Manual <https://mcnp.lanl.gov/pdf_files/TechReport_2017_LANL_LA-UR-17-29981_WernerArmstrongEtAl.pdf>`_
-* `MCNP 6.3 User Manual <https://mcnp.lanl.gov/pdf_files/TechReport_2022_LANL_LA-UR-22-30006Rev.1_KuleszaAdamsEtAl.pdf>`_
```

Comment:
Since we already have automation to update links to the UM using `sphinx.ext.extlinks`, we could alternatively use that here so that it stays in sync, rather than linking to OSTI

Reply:
![image](https://github.com/idaholab/MontePy/assets/12405772/bd475f6a-c5a8-4da3-a9d6-e62483aba934)
I don't like having to include the page. I think having one extra spot where we define the URL should be fine.